royalties-registry/contracts/providers/RoyaltiesProviderArtBlocks.sol

```diff
@@ -23,37 +23,72 @@ contract RoyaltiesProviderArtBlocks is IRoyaltiesProvider, Ownable {
         //gettign artist and additionalPayee royalty part
         (address artistAddress, address additionalPayee, uint256 additionalPayeePercentage, uint256 royaltyFeeByID) = artBlocks.getRoyaltyData(tokenId);
 
+        require(additionalPayeePercentage <= 100 && royaltyFeeByID <= 100, "wrong additionalPayeePercentage value");
+
         //resulting royalties
         LibPart.Part[] memory result;
 
-        if (additionalPayeePercentage > 0){
+        //if no artist royalty
+        if (royaltyFeeByID == 0) {
+            //if artblocksPercentage > 0
+            if (artblocksPercentage > 0) {
+                result = new LibPart.Part[](1);
+
+                //calculating artBLocks part
+                result[0].account = payable(owner());
+                result[0].value = artblocksPercentage;
+            }
+            //if artblocksPercentage = 0 then result is empty
+            return result;
+
+        //if royaltyFeeByID > 0 and  0 < additionalPayeePercentage < 100
+        } else if (additionalPayeePercentage > 0 && additionalPayeePercentage < 100){
```

Comment:
1.Maybe better use: require(additionalPayeePercentage < 100, comment); in Line 26
so here(line 45) you can Use: else if (additionalPayeePercentage > 0) { 
(not need to check additionalPayeePercentage < 100 twice)
2.(line 45) need formatting?:  } else if (additionalPayeePercentage > 0 && additionalPayeePercentage < 100) {

Reply:
additionalPayeePercentage = 100 is a correct value and a specific case when additional payee gets all the royalty

Reply:
formatting fixed